src/openrct2-ui/windows/Tooltip.cpp

```diff
@@ -53,6 +53,10 @@ static int32_t FormatTextForTooltip(const OpenRCT2String& message)
     format_string(_tooltipText, sizeof(_tooltipText), message.str, message.args.Data());
     gCurrentFontSpriteBase = FONT_SPRITE_BASE_MEDIUM;
 
+    OpenRCT2String formattedMessage{ STR_STRING_TOOLTIP, Formatter() };
+    formattedMessage.args.Add<const char*>(_tooltipText);
+    format_string(_tooltipText, sizeof(_tooltipText), formattedMessage.str, formattedMessage.args.Data());
```

Comment:
I am a bit nervous about this reading and writing to the same buffer. I think it is better to use a middle temp buffer for the first format call.